t3/utils/flux.py

```diff
@@ -322,17 +322,21 @@ def run_jsr(gas: ct.Solution,
         dict: The T, P, X, and ROP profiles (values) at a specific time.
```

Comment:
suggestions:

Add an arg to this function with a default value, e.g. 0.01, which indicates the solver to use `network.step()` as along as t < t * tolerance, else use the new implementation here `network.advance()`

Add a test that checks the last time

Reply:
I added this to this branch. For now, the tolerance as a default is 10%. Until tau*10%, the jsr will be solved by step, and the rest of the times by advance.